src/app/shared/components/page-not-found/page-not-found.module.ts

```diff
@@ -0,0 +1,14 @@
+import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
+import { TranslateModule } from '@ngx-translate/core';
+import { RouterModule } from '@angular/router';
+import { PomCommonPipesModule } from '@app/shared/pipes';
+import { PageNotFoundComponent } from './page-not-found.component';
+import { SharedModule } from '../..';
```

Comment:
co konkretnie chcesz tutaj zaimportować ? Wydaje mi się, że cały shared nie jest tutaj Potrzebny

Reply:
breadcrumbs, a ten komponent nie ma modułu

Reply:
Takie importowanie jest niebezpieczne, bo możesz spowodować cykliczną zależność - zazwyczaj foldery wyżej agregują rzeczy ze środka.

Jeśli ten komponent nie jest częścią `SharedModule`, to może nie powinien być w folderze `shared`?

Reply:
Obecność breadcrumbs w komponencie "page not found" chyba nie jest potrzebna. Usunąłem, a razem z nim, zależność od SharedModule.